views/index.handlebars

```diff
@@ -89,7 +89,7 @@
           {{/if}}
         </div>
         <div id="map"></div>
-        <button id="customFullscreenButton" draggable="false" title="Toggle fullscreen view" type="button" style="display: none; background: none rgb(255, 255, 255); border: 0px; margin: 10px; padding: 0px; position: absolute; cursor: pointer; user-select: none; border-radius: 2px; height: 40px; width: 40px; box-shadow: rgba(0, 0, 0, 0.3) 0px 1px 4px -1px; overflow: hidden; top: 0px; right: 0px;">
+        <button id="customFullscreenButton" draggable="false" title="Toggle fullscreen view" type="button" style="background: none rgb(255, 255, 255); border: 0px; margin: 10px; padding: 0px; position: absolute; cursor: pointer; user-select: none; border-radius: 2px; height: 40px; width: 40px; box-shadow: rgba(0, 0, 0, 0.3) 0px 1px 4px -1px; overflow: hidden; top: 0px; right: 0px;">
```

Comment:
does this just need to be added to give.handlebars to fix the embed fullscreen?

Reply:
Yes, I think so. I just wasn't sure if the fullscreen was wanted on the embed.